Bottom: 0a346b072584964eeb4e3a84449a27cd5a0417d9
Top:    b967c99a1264a3953e5343a78eb32be3ab16c0b9
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-05-06 14:02:49 +0530

Refresh of sched-idle-add-debugging-bits

---

diff --git a/kernel/sched/idle.c b/kernel/sched/idle.c
index 5834273378b6..6581613189d9 100644
--- a/kernel/sched/idle.c
+++ b/kernel/sched/idle.c
@@ -233,7 +233,7 @@ static void do_idle(void)
 	int cpu = smp_processor_id();
 	int pm_disabled = atomic_read(&per_cpu(nr_lat_sensitive, cpu));
 
-	if (pm_disabled)
+	if (pm_disabled < 0)
 		pr_info("Inconsistent value of nr_lat_sensitive counter\n");
 
 	/*
